lib/src/callable/built_in.dart

```diff
@@ -21,6 +21,8 @@ final class BuiltInCallable implements Callable, AsyncBuiltInCallable {
   /// The overloads declared for this callable.
   final List<(ArgumentDeclaration, Callback)> _overloads;
 
+  bool hasContent = false;
```

Comment:
Not super happy with adding this property on the main `BuiltInCallable` class, but it is necessary in `meta.accepts-content` to know whether the builtin mixin callable hasContent because of `meta.apply`, which does accept content in some scenarios.

Reply:
I suppose one alternative is to have `meta.apply` not actually return true for `meta.accepts-content`.

Reply:
From a strict reading of the spec, the question `meta.accepts-content` asks is "does the body of this mixin have a `@content` rule?" The body of `meta.apply` doesn't exist, so the answer is "no". I think it's safe to just return "no" here as well. (It's also the case that _many_ invocations of `meta.apply` won't actually accept a content block in practice.)